lib/stacker/resolver.rb

```diff
@@ -8,22 +8,32 @@ def initialize region, parameters
     end
 
     def dependencies
-      @dependencies ||= parameters.select { |_, value|
+      @dependencies ||= parameters.values.flatten.select { |value|
```

Comment:
@kainosnoema I should have written it this way in the first place. I just pulled the flatten ahead of the reference mapping.